spinnaker_components/ensemble/ensemble.h

```diff
@@ -65,10 +58,11 @@ typedef struct ensemble_parameters {
   uint machine_timestep;   //!< Machine time step  / useconds
 
   uint t_ref;              //!< Refractory period \f$\tau_{ref} - 1\f$ / steps
-  value_t dt_over_t_rc;    //!< \f$\frac{dt}{\tau_{rc}}\$
+  value_t exp_dt_over_t_rc;    //!< \f$-\exp{\frac{dt}{\tau_{rc}}}\$
 
   current_t *i_bias;        //!< Population biases \f$1 \times N\f$
-  neuron_status_t *status;  //!< Neuron status
+  value_t *neuron_voltage;  //!< Neuron voltages
+  uint8_t *neuron_refractory;  //!< Refractory states
```

Comment:
Does this change affect the formula for determining how many neuron there can be on a core?